src/snowcli/cli/snowpark_shared.py

```diff
@@ -17,17 +18,41 @@
     help="Whether to download non-Anaconda packages from PyPi. Valid values include: `yes`, `no`, `ask`. Default: `no`.",
     callback=yes_no_ask_callback,
 )
+
 PackageNativeLibrariesOption = typer.Option(
     "ask",
     help="When using packages from PyPi, whether to allow native libraries. Valid values include: `yes`, `no`, `ask`. Default: `no`.",
     callback=yes_no_ask_callback,
 )
-CheckAnacondaForPyPiDependancies: bool = typer.Option(
+
+CheckAnacondaForPyPiDependencies: bool = typer.Option(
     True,
     "--check-anaconda-for-pypi-deps/--no-check-anaconda-for-pypi-deps",
     "-a",
     help="""Whether to check if any of missing Anaconda packages dependencies can be imported directly from Anaconda. Valid values include: `true`, `false`, Default: `true`.""",
 )
+
+LikeOption = typer.Option(
+    "%%",
+    "--like",
+    "-l",
+    help='Regular expression for filtering the functions by name. For example, `list --file "my%"` lists all functions in the **dev** (default) environment that begin with “my”.',
```

Comment:
So, is it `--file` or `--like` or both? It seems a bit unclear to me

Reply:
😄  It should be `--like`